module/input/Microphone/src/Microphone.cpp

```diff
@@ -0,0 +1,380 @@
+#include <alsa/asoundlib.h>
+#include <spawn.h>
+#include <fmt/format.h>
+#include "Microphone.hpp"
+
+#include "extension/Configuration.hpp"
+#include "json.h"
```

Comment:
@KipHamiltons, I'm just wondering about this include. It's a single header file located at: https://github.com/sheredom/json.h
How would you recommend adding it as a dependency?

Reply:
Same as we do with the catch header [here](https://github.com/NUbots/NUbots/blob/main/docker/Dockerfile#L220)

then it'll be in a folder such that you should be able to do `#include <json.h>`, I am pretty sure